lib/modules/landing/landing_presenter.dart

```diff
@@ -13,4 +26,34 @@ class LandingPresenterImpl extends LandingPresenter
 
   @override
   final animationDidFinish = BehaviorSubject<void>();
```

Comment:
here @markgravity; not sure then how come it recognize as bool when we zip the stream 😂

Reply:
or maybe I understand it incorrectly 😅 maybe first is the 2nd stream output type, and then 2nd is the final output type 😅

Reply:
sorry @sleepylee, I missed your comments here 😁 
I think they make me confuse same as you, that why I try to use my extension to skip this confusing step lol
```dart
[animationDidFinish, authenticationDidValidate].zip().listen((data) {}) // data is List<dynamic>
```